### ISSUE TITLE
Implement direct link to images

### ISSUE THREAD

**Issue by mdolr** [mdolr/survol#109]
When a direct link to an imgur image is hovered it should be previewed, not just for imgur but any host.

**Comment by debPhoenix** [comment]
Hi! Can I work on it?

**Comment by mdolr** [comment]
Yes of course, I'm not really sure how you should go about implementing this, I'd say doing it in `js/templates/base.js` seems like a good option

**Comment by mdolr** [comment]
@debPhoenix Please don't ask to work on something without even looking at what should be done and then delete your comments when you realize you can't do it (20 minutes after asking to work on the issue lol) without saying anything :) This is not an easy hacktoberfest repository :)

**Comment by Dxuian** [comment]
is this repo still open for contribution or deprecating  ?

**Comment by mdolr** [comment]
it's abandoned @Dxuian
